packages/salesforcedx-sobjects-faux-generator/test/integration/sObjectDescribe.test.ts

```diff
@@ -25,40 +26,48 @@ const SIMPLE_OBJECT_DIR = path.join(
   'objects'
 );
 
-//const util = new TestUtil();
 const sobjectdescribe = new SObjectDescribe();
 
 // tslint:disable:no-unused-expression
 describe('Fetch sObjects', function() {
+  // tslint:disable-next-line:no-invalid-this
+  this.timeout(60000);
   let username: string;
 
   before(async function() {
+    await util.createSFDXProject(PROJECT_NAME);
+    console.log('process: ' + process.cwd());
+    util.createCIKey(keyLocation);
+    console.log('key: ' + keyLocation);
+    username = await util.createScratchOrg(PROJECT_NAME);
+
     const sourceFolder = path.join(
       __dirname,
       '..',
       '..',
       '..',
       SIMPLE_OBJECT_DIR
     );
-    const permSetName = 'AllowRead';
-    await util.createSFDXProject(PROJECT_NAME);
-    username = await util.createScratchOrg(PROJECT_NAME);
-
     await util.push(sourceFolder, PROJECT_NAME, username);
+
+    const permSetName = 'AllowRead';
     const permissionSetId = await util.createPermissionSet(
       permSetName,
       username
     );
+
     await util.createFieldPermissions(
       permissionSetId,
       CUSTOM_OBJECT_NAME,
       CUSTOM_FIELD_FULLNAME,
       username
     );
+
     await util.assignPermissionSet(permSetName, username);
   });
 
   after(function() {
+    util.deleteCIKey(keyLocation);
```

Comment:
Not strictly required since the key will also get deleted through the call below.